src/c/vwf.c

```diff
@@ -841,6 +841,60 @@ int print_string(byte *str, int x, int y)
     return (charCount & 0xFFFF) | (totalWidth << 16);
 }
 
+// Edited version which recognizes the 5F FF code
+// Returns: ____XXXX = number of characters printed
+//          XXXX____ = number of pixels printed
+// x, y: pixels
+int print_string_edited(byte *str, int x, int y)
+{
+    if (str == NULL)
+        return 0;
+
+    byte chr;
+    int initial_x = x;
+    int charCount = 0;
+
+    while (!(str[1] == 0xFF && str[0] == 0x00))
+    {
+        if(str[1] != 0xFF)
+        {
+            x += print_character(decode_character(*str++), x, y);
+            charCount++;
+        }
+        else if(str[0] == 0x5F)
+        {
+            x += str[2];
+            str += 3;
+        }
```

Comment:
Minor thing but this acts more like 60 FF (CUSTOMCC_ADD_X) than 5F FF (CUSTOMCC_SET_X). It might be a little confusing to have 5F mean a different thing here.

Reply:
Updated. To make it generic, it can't (in theory) access the window data to base itself off of, so initial_x is a good compromise.